tlslite/utils/format_input.py

```diff
@@ -0,0 +1,21 @@
+# Authors:
+#   Hubert Kario
```

Comment:
but you've refactored it, and you've created the file so your authorship should be there